webpack.mix.js

```diff
@@ -11,21 +11,14 @@
  * SPDX-License-Identifier: EPL-2.0
 */
 
-mix = require('laravel-mix');
-require('laravel-mix-transpile-node-modules')
-mix.transpileNodeModules(['eclipsefdn-solstice-assets']);
+require('./node_modules/eclipsefdn-solstice-assets/webpack-solstice-assets.mix.js');
+let mix = require('laravel-mix');
+mix.EclipseFdnSolsticeAssets();
+
 mix.options({uglify: {uglifyOptions: {compress: false, output: {comments: true}}}});
```

Comment:
I guess we could also remove mix.options here

Reply:
good catch!